report/report.go

```diff
@@ -209,4 +209,6 @@ const (
 	HostNodeID = "host_node_id"
 	// ProbeID is the random ID of the probe which generated the specific node.
 	ProbeID = "probe_id"
+	// ControlProbeID is the random ID of the probe which controls the specific node.
+	ControlProbeID = "control_probe_id"
```

Comment:
I don't think we should add another probe id (ControlProbeID ) to the metadata.  I think we should be more specific about what ProbeID is, and make sure its added correctly.